vendor/modules.txt

```diff
@@ -216,6 +216,8 @@ google.golang.org/protobuf/types/known/timestamppb
 # gopkg.in/inf.v0 v0.9.1
 ## explicit
 gopkg.in/inf.v0
+# gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
```

Comment:
It looks like you added only 2 commented stings in to the file. Are you sure it would fix the root issue ?

Reply:
It affected more files, I hope it will help.
If it won't, we will revert this one.

Reply:
But you need to updated dependencies right ? Looks like the old yaml dep is still there